README.md

```diff
@@ -1,8 +1,31 @@
 # Redux-cycles
+
+<div align="center">
+  <img src="logo.png" alt="Cycle.js + Redux = <3" />
```

Comment:
Maybe even simplify a bit and use a blue heart to be more in line, color wise, with Redux and Cycle logos:

![redux-cycles-alt2](https://cloud.githubusercontent.com/assets/315596/22520005/b0d0a3a0-e8c3-11e6-84ef-481398865b95.png)